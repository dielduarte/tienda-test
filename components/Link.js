import styled from 'styled-components';

const Link = styled.a`
  color: ${props => props.theme.lightBlue};
  text-decoration: none;
`;

export default Link;
