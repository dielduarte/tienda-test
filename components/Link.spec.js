import { shallow } from 'enzyme';
import 'jest-styled-components';

import Link from './Link';

describe('Link', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Link>meu produto</Link>);
    expect(wrapper).toMatchSnapshot();
  });
});
