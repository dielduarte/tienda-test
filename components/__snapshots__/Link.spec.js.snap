// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render correctly 1`] = `
.c0 {
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  className="c0"
>
  meu produto
</a>
`;
